uint32  ch1
uint32  ch2
uint32  ch3
uint32  ch4
uint32  ch5
uint32  ch6
uint32  ch7
uint32  ch8
uint32  ch9
uint32  ch10
uint32  ch11
uint32  ch12
uint32  ch13
uint32  ch14
uint32  ch15
uint32  ch16
uint32  ch17
uint32  ch18
uint8   frame_lost
uint8   failsafe
uint8   reserve